robotiq_c_model_control/src/robotiq_c_model_control/gripper_driver.py

```diff
@@ -0,0 +1,255 @@
+"""
+Copyright 2018 by READY Robotics Corporation.
+All rights reserved. No person may copy, distribute, publicly display, create derivative works from or otherwise
+use or modify this software without first obtaining a license from the READY Robotics Corporation.
+"""
+import ready_logging
+import rospy
+from robotiq_c_model_control.base_c_model import BaseCModel
+from robotiq_c_model_control.constants import (
+    MAX_GRIPPER_COUNT,
+    MODBUS,
+    ORIGINAL,
+    VALID_DEVICE_IDS
+)
+from robotiq_c_model_control.robotiq_gripper import RobotiqGripper
+from robotiq_c_model_control.util import gripper_name_generator
+from robotiq_modbus_rtu.com_modbus_rtu import Communication
+from robotiq_modbus_rtu.single_com_modbus_rtu import (
+    SingleCommunication,
+    SingleCommClient
+)
+
+
+@ready_logging.logged
+class GripperDriver(object):
+    """ Driver for running multiple Robotiq grippers on the same MODBUS. """
+    @staticmethod
+    def activate(grippers):
+        """ Activate all the provided grippers simultaneously. """
+        # Start gripper threads
+        for gripper in grippers:
+            gripper.start()
+
+        # Wait for all gripper resets to complete.
+        for gripper in grippers:
+            while gripper.resetting:
+                if rospy.is_shutdown():
+                    break
```

Comment:
```suggestion
                    return
```

Reply:
I like break here in case clean-up needs added after the for-loop. How do you feel about that?

Reply:
Once you call shutdown, everything is hosed. You could easily use the rospy.on_shutdown method to do the cleanup.

Reply:
I see your point though and either is fine